src/plugins/intel_gpu/include/intel_gpu/runtime/memory_pool.hpp

```diff
@@ -96,7 +99,7 @@ class memory_pool {
 
     std::multimap<uint64_t, memory_record> _non_padded_pool;
     std::map<layout, std::list<memory_record>, padded_pool_comparer> _padded_pool;
-    std::multimap<uint64_t, memory_record> _no_reusable_pool;
+    std::multimap<uint64_t, memory_record> _across_network_pool;
```

Comment:
Why did you change the name? Actually "across network reuse" is not considered to be used for now. (Deactivated) 
(In my thouhgt we even need to remove the legacy non used scenario of "reuse b/w network" to remove mislead ...) Changing name will cause more confusion.

Reply:
Done.